changelog.md

```diff
@@ -3,6 +3,44 @@ layout: default
 css_id: changelog
 ---
 
-# Changelog
 
-## Changes since v1.0.0
+The changes made to the Uptane Standard since its initial release on July 31, 2019, have principally addressed issues of style, clarity, and the resolution of inconsistencies. As a result, the majority of text edits and additions seek to correct wording in the original text that could potentially be misleading. 
+
+The following is a brief summary of those edits:
+
+- Established a style guide to impose consistency in spelling, capitalization of roles and repository names, and use of punctuation. 
+
+- Specified a policy for linking to the Standard or any specific portion of it. Any links to the Standard from other documents should point to the latest released version, and should  link by section name, not number, as the numbers tend to change more than the names.
+
+- Established a document archive policy to add a stable copy of each version of the Standard to the repository, starting with the initial IEEE/ISTO V.1.0.0 document.
+
+- Added “Compromise of the packaged software, such as malware embedded in a trusted package” to the list of what is "Out of Scope" for the Standard.
+
+- Corrected/changed several numbering references in the full verification process and “Step 0” in the procedure for checking root metadata.
+
+- Presented the option to use a counter, instead of a nonce in the ECU Version Report, and specified the purpose of the nonce in the step-by-step instructions for preparing this report. 
```

Comment:
```suggestion
- Presented the option to use a counter (instead of a nonce) in the ECU Version Report, and specified the purpose of the nonce in the step-by-step instructions for preparing this report. 
```